.prettierrc

```diff
@@ -1,3 +1,4 @@
 {
+  "parser": "babel",
```

Comment:
Why was this necessary?

Reply:
As far as I remember it's a required option. I can double check though...

Reply:
I don't think it's required, cause it's working fine today.